### ISSUE TITLE
Problem z wymiarami przesyłek DHL

### ISSUE THREAD

**Issue by shivson1337** [allegro/allegro-api#9282]
Występuje problem: 
 - [x] z zasobem REST API / with a REST API resource
 - [x] ze środowiskiem produkcyjnym / with the production environment

:beetle: **Opis / Description**
Od wczoraj (tj. 25.06.2024) pojawiły się błędy przy nadawaniu przesyłek do punktów DHL (np. żabka).
Wymiary, które były akceptowalne dotychczas przestały być akceptowane. Nie zauważyliśmy nigdzie zmian w tej kwestii.

Czy możemy prosić o podanie aktualnych dopuszczalnych wymiarów przesyłek dla tego przewoźnika?

trace-id: 221a3cf7cd25b5c7

**Comment by PrzemyslawLukanowski** [comment]
Weryfikujemy ten temat, wstępnie wygląda to na problem po naszej stronie, ponieważ odrzuciliśmy poprawnie zdefiniowaną szerokość przesyłki. Wrócę z informacją, gdy uda nam się ustalić więcej szczegółów.

**Comment by PrzemyslawLukanowski** [comment]
Możesz powtórzyć próbę utworzenia paczki, problem już nie powinien występować.

**Comment by shivson1337** [comment]
Wszystko wróciło do normy.

**Comment by PrzemyslawLukanowski** [comment]
Dziękuję za potwierdzenie i przepraszam za niedogodności.

**Comment by kulisz** [comment]
@PrzemyslawLukanowski chyba nadal występuje jakiś problem, nadajemy paczkę o wymiarach 
`:packages=>[{:type=>"PACKAGE", :length=>{:value=>13, :unit=>"CENTIMETER"}, :width=>{:value=>18, :unit=>"CENTIMETER"}, :height=>{:value=>40, :unit=>"CENTIMETER"}, :weight=>{:value=>"1", :unit=>"KILOGRAMS"}}],`
czyli mieści się w 
<img width="602" alt="image" src="https://github.com/allegro/allegro-api/assets/232743/3736af1c-180e-4382-9dfb-3e5256bfb53a">
a i tak dostajemy błąd 

> Komunikat błędu: Przekroczono maksymalną dopuszczalną wysokość

**Comment by PrzemyslawLukanowski** [comment]
@kulisz Mógłbym jeszcze przesłać commandId lub trace-id tej operacji?

**Comment by kulisz** [comment]
@PrzemyslawLukanowski przesyłka utworzona z commandId f47b6f66-beb2-447d-8184-7f809ecce4ab

**Comment by PrzemyslawLukanowski** [comment]
Dla odbioru w punkcie maksymalna wysokość to 38cm - https://help.allegro.com/sell/pl/a/allegro-dhl-z-wysylka-do-punktow-odbioru-i-automatow-paczkowych-informacje-dla-sprzedajacych-9g0zY54o0S6?marketplaceId=allegro-pl

> 
> Maksymalne wymiary oraz waga przesyłek, które nadasz metodą Allegro Automat DHL BOX 24/7 do automatu paczkowego i metodą Allegro Odbiór w Punkcie DHL do punktu odbioru:
> - waga – 25 kg
> - długość – 64 cm
> - szerokość – 41 cm
> - wysokość – 38 cm
> - obwód (długość + 2x szerokość + 2x wysokość) – 220 cm.

**Comment by kulisz** [comment]
@PrzemyslawLukanowski https://help.allegro.com/sell/pl/a/nowosci-i-zmiany-dla-sprzedajacych-w-czerwcu-lipcu-i-wrzesniu-Xx4ROByz5tY#zmniejszymy-maksymalne-wymiary-paczek-allegro-odbior-w-punkcie-dhl-i-zmienimy-nazwe-metody-dostawy-do-automatow-paczkowych stąd brałem informację na temat przesyłki gdzie nie było określonej kolejności parametrów a więc wzorowałem się na kolejności z obwodu, na przyszłość warto dodać informację o kolejności podawanych parametrów

**Comment by PrzemyslawLukanowski** [comment]
Oczywiście, przekażę informację do zespołu odpowiedzialnego za stronę. Dziękuję za sugestię.
